<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QLauncherClass</class>
 <widget class="QDialog" name="QLauncherClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QLauncher</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTreeView" name="treeConnectionList">
       <property name="palette">
        <palette>
         <active>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>240</red>
             <green>240</green>
             <blue>240</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="showDropIndicator" stdset="0">
        <bool>false</bool>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTreeView" name="treeProgramList">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QListWidget" name="listLog">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="btnAdd">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>추가</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnDelete">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>삭제</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnOn">
         <property name="text">
          <string>On</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnOnAll">
         <property name="text">
          <string>On All</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnOff">
         <property name="text">
          <string>Off</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnOffAll">
         <property name="text">
          <string>Off All</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnPowerOff">
         <property name="text">
          <string>전원 끄기</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnReboot">
         <property name="text">
          <string>재부팅</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QCheckBox" name="checkServerList">
         <property name="text">
          <string>Server List</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnServerConn">
         <property name="text">
          <string>서버 접속</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="tableServerList"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QCheckBox" name="checkMacList">
         <property name="text">
          <string>WOL Mac address List</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnPower">
         <property name="text">
          <string>전원 켜기</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="tableMac"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="edTargetIp"/>
     </item>
     <item>
      <widget class="QLineEdit" name="edTargetPort"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnSaveConf">
       <property name="text">
        <string>ip설정저장</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qlauncher.qrc"/>
 </resources>
 <connections/>
</ui>
